### ISSUE TITLE
Profile export failed

### ISSUE THREAD

**Issue by mkevenaar** [spectresystems/ghostly#76]
**Ghostly version**
1.10.0 (x64)

**Windows 10 version**
1909

**Describe the bug**
I signed out and logged back in to enable gists, after I tried to sync I got an error message.

**To Reproduce**
1. Go to Accounts
2. Click on Account (menu)
3. Click on Profiles
4. Click on Export profile
5. Click on Create new profile
6. Click on Export
7. I used the default name (hostname) and clicked on Create

**Expected behavior**
The profile to be exported without an error message.

**Screenshots** (optional)
Error message:
![image](https://user-images.githubusercontent.com/834643/86891411-146b6f00-c0ff-11ea-9195-207656c46810.png)

Ghostly permissions:
![image](https://user-images.githubusercontent.com/834643/86893536-0ec35880-c102-11ea-8487-fc1165db3dae.png)

**Additional context** (optional)
```
{"@t":"2020-07-08T07:39:32.2388105Z","@mt":"An error occured while contacting GitHub.","@l":"Error","@x":"Octokit.NotFoundException: Not Found\r\n   at Octokit.Connection.HandleErrors(IResponse) + 0x8f\r\n   at Octokit.Connection.<RunRequest>d__60.MoveNext() + 0x80e\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17\r\n   at Octokit.Connection.<Run>d__59`1.MoveNext() + 0x112\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17\r\n   at Octokit.ApiConnection.<Post>d__23`1.MoveNext() + 0x1d9\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17\r\n   at Ghostly.GitHub.Legacy.Octokit.GitHubGateway.<Execute>d__11`1.MoveNext() + 0x16a\r\n\r\n{\"message\":\"Not Found\",\"documentation_url\":\"https://developer.github.com/v3/gists/#create-a-gist\"}","Session":[DELETED]}
```

**Comment by mkevenaar** [comment]
I think this is a way to reproduce:

* Add an account without gist permissions. (probably with an older version of Ghostly)
* Remove that account
* Add it again with gist permissions
* Try to export a profile.

This is solved after a restart of Ghostly.

**Comment by patriksvensson** [comment]
@mkevenaar Thanks! I've managed to reproduce this and will publish a fix as soon as possible 👍

**Comment by patriksvensson** [comment]
This bug has been fixed in version `1.11.0` and will be available in a couple of hours in the Microsoft Store.
Thank you for reporting this! 👍

https://spectresystems.se/ghostly/docs/release-notes
